packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -94,7 +100,7 @@ function LoginForm({
           <ClickableText
             type="button"
             className="flex-1 underline grow-[2] btn-primary"
-            onClick={onForgotPassword}
+            onClick={() => onForgotPassword(innerEmail)}
```

Comment:
I think if you use value still, we can leave this was it was?

Reply:
The `email` coming from props here will be undefined until the form is submitted, so we need to keep track of it separately for the forgot password handler